README.md

```diff
@@ -16,13 +19,19 @@ So play DJ on your phone while listening on Mycroft!
 This Skill uses two different methods of authentication. Both need to be filled in correctly for the **Skill** to function correctly.
 
 #### API connection to your Spotify account
-After installing `mycroft-spotify`, in your [Skill
-settings for Spotify](https://home.mycroft.ai/#/skill) in home.mycroft.ai you will see settings for the Spotify Skill. You will see a username and password field and a 'Connect' button. Ignore the username and password field for now, and click the 'Connect' button. You will be prompted to log in to Spotify, and to authorize Mycroft AI to use your Spotify account using OAuth. This allows Mycroft access to your account details such as Playlists.
+After installing `mycroft-spotify`, from the mycroft-core folder run the auth.py script in the mycroft-spotify folder
+
+```
+source venv-activate-sh
+python skills/mycroft-spotify/auth.py
```

Comment:
in my installation, it is named:
```
(.venv) pi@picroft:~/mycroft-core $ python skills/mycroft-spotify/auth.py
python: can't open file 'skills/mycroft-spotify/auth.py': [Errno 2] No such file or directory
(.venv) pi@picroft:~/mycroft-core $ ll skills/mycroft-spotify.forslund/
total 128
```
?

Reply:
Ah yes, forgot the `.forslund` part. will fix that

Possibly we should point it to the full path (though this may change later) `/opt/mycroft/skills/mycroft-spotify.forslund`

Reply:
Ya we could do that; (absolute path vs. relative path) :shrug: 
(the instructions say to run script from XX directory which makes relative paths "safe-ish" ;o)

Reply:
this was fixed :heavy_check_mark: